apis/v1alpha1/generator.yaml

```diff
@@ -17,12 +18,131 @@ ignore:
+      HealthCheckId:
+        from:
+          operation: ChangeResourceRecordSets
+          path: ChangeBatch.Changes.ResourceRecordSet.HealthCheckId
+      # Changing this value after a CR has been created could result in orphaned record sets
+      HostedZoneId:
+        references:
+          resource: HostedZone
+          path: Status.ID
+        is_required: true
+        is_immutable: true
+      ID:
+        is_primary_key: true
+        documentation: "ChangeID is used as a unique identifier"
+      MultiValueAnswer:
+        from:
+          operation: ChangeResourceRecordSets
+          path: ChangeBatch.Changes.ResourceRecordSet.MultiValueAnswer
+      # Changing this value after a CR has been created could result in orphaned record sets
+      Name:
+        from:
+          operation: ChangeResourceRecordSets
+          path: ChangeBatch.Changes.ResourceRecordSet.Name
+        is_required: true
+        is_immutable: true
+      # Changing this value after a CR has been created could result in orphaned record sets
+      RecordType:
+        from:
+          operation: ChangeResourceRecordSets
+          path: ChangeBatch.Changes.ResourceRecordSet.Type
+        is_required: true
+        is_immutable: true
+      Region:
+        from:
+          operation: ChangeResourceRecordSets
+          path: ChangeBatch.Changes.ResourceRecordSet.Region
+      ResourceRecords:
+        from:
+          operation: ChangeResourceRecordSets
+          path: ChangeBatch.Changes.ResourceRecordSet.ResourceRecords
+      # Changing this value after a CR has been created could result in orphaned record sets
+      SetIdentifier:
+        from:
+          operation: ChangeResourceRecordSets
+          path: ChangeBatch.Changes.ResourceRecordSet.SetIdentifier
+        is_immutable: true
+      Status:
+        print:
+          name: STATUS
+      TTL:
+        from:
+          operation: ChangeResourceRecordSets
+          path: ChangeBatch.Changes.ResourceRecordSet.TTL
+      Weight:
+        from:
+          operation: ChangeResourceRecordSets
+          path: ChangeBatch.Changes.ResourceRecordSet.Weight
+    hooks:
+      sdk_create_post_build_request:
+        template_path: hooks/record_set/sdk_create_post_build_request.go.tpl
+      sdk_delete_post_build_request:
+        template_path: hooks/record_set/sdk_delete_post_build_request.go.tpl
+      sdk_read_many_post_build_request:
+        template_path: hooks/record_set/sdk_read_many_post_build_request.go.tpl
+      sdk_read_many_pre_set_output:
+        template_path: hooks/record_set/sdk_read_many_pre_set_output.go.tpl
+      sdk_read_many_post_set_output:
+        template_path: hooks/record_set/sdk_read_many_post_set_output.go.tpl
+    list_operation:
+      match_fields:
+      - HostedZoneId
+      - Name
+      - SetIdentifier
+    # Requeue successful reconciliations to update the status of the record set
+    reconcile:
+      requeue_on_success_seconds: 60
```

Comment:
Now that we support [drift remediation configuration via flags/helm values](https://aws-controllers-k8s.github.io/community/docs/user-docs/drift-recovery/), we can drop this.